bonfire/utils.py

```diff
@@ -555,28 +551,29 @@ def hms_to_seconds(s):
 
 
 @lru_cache(maxsize=None)
-def check_hostname(hostname):
+def __check_connection(hostname, port=443, timeout=5):
     """
     Check connection makes sure a connection is available to a given hostname.
 
-    Function is cached so that we only check a hostname once.
+    Function is cached so that we onl
+    y check a hostname once.
```

Comment:
Again being picky here but can we fix this mid-word newline too? :)

Reply:
non picky, this is a mistake from the editor formatting, my bad. Thanks for spotting this.